src/components/Trends/TrendsPanel.jsx

```diff
@@ -164,8 +156,6 @@ const mapStateToProps = state => {
   const {
     company: companyFilters,
     dateInterval,
-    issue: issueFilters,
```

Comment:
it's funny that this thing wasn't being used for anything.  I did a doubletake once i looked at why focusData was passing in issueFilters into the function.  Probably some legacy code from explorer.  🤷